packages/hooks/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from '@jest/types'
+
+import base from '../../jest.config.base'
+
+import packageJson from './package.json'
+
+const config: Config.InitialOptions = {
```

Comment:
Could you at a placeholder for tests, I think this will fail without any tests. See here how to add a todo test: https://github.com/hyperledger/aries-framework-javascript-ext/blob/main/packages/redux-store/tests/index.test.ts